src/components/TodoListItem.jsx

```diff
@@ -0,0 +1,27 @@
+require ('../styles/todoList.css');
+
+import React from 'react';
+
+class TodoListItem extends React.Component {
+
+  constructor(props) {
+    super(props);
+      this.updateState = this.updateState.bind(this);
+    };
+
+    updateState() {
+      this.setState({
+        line: {
+            textDecoration: "line-through"
```

Comment:
zamieniłbym to raczej na jakąś flagę `completed` i na tej podstawie w `render` określał `style`, a nie trzymał sam `style` w `state`